### DISCUSSION TITLE
Support for the new css nesting syntax

### DISCUSSION THREAD

**Discussion by ahmedragab20** [vuejs/rfcs#638] [General]
for your information, it works when I save the changes.

![image](https://github.com/vuejs/rfcs/assets/71534456/4e9235bc-d896-42c8-8009-d42d17388b28)

**Comment by ShGKme** [comment]
Vue does nothing with support or not support CSS Nesting. It works in SFC, with scoped, with CSS Modules.

The issue on the screenshot is probably from VSCode: https://github.com/microsoft/vscode-css/issues/9

**Reply by ahmedragab20** [reply]
ah well, understood 🤝🏻
